diff --git a/node_modules/@mui/material/styles/createPalette.d.ts b/node_modules/@mui/material/styles/createPalette.d.ts
index 5e28388..bea9fdf 100644
--- a/node_modules/@mui/material/styles/createPalette.d.ts
+++ b/node_modules/@mui/material/styles/createPalette.d.ts
@@ -89,7 +89,9 @@ export interface Palette {
   warning: PaletteColor;
   info: PaletteColor;
   success: PaletteColor;
+  custom: PaletteColor;
   grey: Color;
+  grey99: PaletteColor;
   text: TypeText;
   divider: TypeDivider;
   action: TypeAction;
@@ -114,6 +116,8 @@ export interface PaletteOptions {
   warning?: PaletteColorOptions;
   info?: PaletteColorOptions;
   success?: PaletteColorOptions;
+  custom?: PaletteColorOptions;
+  grey99?: PaletteColorOptions;
   mode?: PaletteMode;
   tonalOffset?: PaletteTonalOffset;
   contrastThreshold?: number;
diff --git a/node_modules/@mui/material/styles/createTypography.d.ts b/node_modules/@mui/material/styles/createTypography.d.ts
index d45b25a..12ecd88 100644
--- a/node_modules/@mui/material/styles/createTypography.d.ts
+++ b/node_modules/@mui/material/styles/createTypography.d.ts
@@ -15,6 +15,9 @@ export type Variant =
   | 'body2'
   | 'caption'
   | 'button'
+  | 'custom'
+  | 'table1'
+  | 'table2'
   | 'overline';
 
 export interface FontStyle