diff --git a/node_modules/@mui/material/styles/createTypography.d.ts b/node_modules/@mui/material/styles/createTypography.d.ts
index d45b25a..da75533 100644
--- a/node_modules/@mui/material/styles/createTypography.d.ts
+++ b/node_modules/@mui/material/styles/createTypography.d.ts
@@ -15,7 +15,10 @@ export type Variant =
   | 'body2'
   | 'caption'
   | 'button'
-  | 'overline';
+  | 'overline'
+  | 'table1'
+  | 'table2'
+  | 'custom';
 
 export interface FontStyle
   extends Required<{
